src/api/getScores.js

```diff
@@ -0,0 +1,11 @@
+import { BASE_URL, GAME_ID } from "./constants.js";
+
+export const getScores = async () => {
+  try {
+    const response = await fetch(`${BASE_URL}/games/${GAME_ID}/scores`);
+    const { result } = await response.json();
+    return result;
+  } catch (error) {
+    console.error(error);
+  }
+};
```

Comment:
- [ ] [OPTIONAL] Kindly sort the scores, either alphabetically or numerically. This will ensure a better scoreboard or leaderboard viewing. check [this link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort) to see the know how 😉.